helm-charts/splunk-otel-collector/templates/NOTES.txt

```diff
@@ -82,9 +82,3 @@ Splunk OpenTelemetry Collector is installed and configured to send data to Splun
   - Some libraries may be enabled by default. For current status, see: https://github.com/open-telemetry/opentelemetry-operator#controlling-instrumentation-capabilities
   - Splunk provides best-effort support for native OpenTelemetry libraries, and full support for Splunk library distributions. For used libraries, refer to the values.yaml under "operator.instrumentation.spec".
 {{- end }}
-{{- if and (not (eq (toString .Values.networkExplorer) "<nil>")) .Values.networkExplorer.enabled }}
-{{ fail "Network explorer is not part of this helm chart anymore. Please follow https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md#0870-to-0880 to replace it with the upstream OpenTelemetry eBPF helm chart." }}
-{{- end }}
-{{- if not (eq (toString .Values.networkExplorer) "<nil>")}}
-[WARNING] `networkExplorer` option is no longer supported. Please remove it from your custom values.yaml.
-{{- end }}
```

Comment:
Please keep the second one. It doesn't cause any problems but incentivise users to drop that option